client/js/qr.ts

```diff
@@ -1,17 +1,23 @@
-let typeNumber = 0;
-let errorCorrectionLevel = 'H';
 declare var qrcode: any;
```

Comment:
Use the `@types/qrcode-generator` npm package (`npm install --save-dev`) to get the right types for this package.

Reply:
@illegalprime that package is actually out of date with the qr code library we're using, so I just defined the interface myself